src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -115,6 +121,13 @@ case class Config(
     }
 }
 
+object Config {
+  // the header tag is assumed to be an invalid option to Gobra's cli
+  val headerTag = "gobra"
+  val header = """##\(%s\)""".format(headerTag).r
```

Comment:
here we are assuming anyway that one cannot combine it with other infile configs (except when splitting them into separate comments)